apps/webhook_listeners/lib/BackgroundJobs/WebhookCall.php

```diff
@@ -92,4 +95,31 @@ protected function run($argument): void {
 			$this->logger->error('Webhook(' . $webhookId . ') call failed: ' . $e->getMessage(), ['exception' => $e]);
 		}
 	}
+
+	private function getTokens($webhookListener, $triggerUserId): array {
```

Comment:
```suggestion
	private function getTokens(\OCA\WebhookListeners\Db\WebhookListener $webhookListener, ?string $triggerUserId): array {
```
Better to import `OCA\WebhookListeners\Db\WebhookListener`.